doc/nrf/releases/release-notes-latest.rst

```diff
@@ -694,17 +706,20 @@ The following list summarizes the most important changes inherited from upstream
   * Updated LVGL to v7.6.1.
   * Updated libmetal and openamp to v2020.10.
   * Updated nrfx in hal-nordic to version 2.4.0.
+  * Moved the nrfx glue code from the hal_nordic module repository to the main zephyr tree.
```

Comment:
```suggestion
  * Moved the nrfx glue code from the hal_nordic module repository to the main Zephyr tree.
```

Reply:
@ru-fu the upstream Zephyr project convention is to use lowercase 'z' when referring to the Git repository whose name is "zephyr", as is being done here ("tree" means "Git repository"). Does NCS have the opposite convention? That should probably be reversed if so, IMO.

Reply:
In NCS context, the repo is usually sdk-zephyr, so no need for this kind of convention. I was considering if this refers to "Zephyr" in general or to the repo and figured if it was the repository, it should say repository and not tree. ;)
But I'm okay with leaving it lowercase.

Reply:
OK, I understand.

The repository's name in west.yml is still 'zephyr', and its path is 'zephyr', so the fact that its current URL contains 'sdk-zephyr' might not be something users are familiar with, I think. So I think that between that and keeping convention with the  zephyr docs, a lowercase 'z' may be best since you are agreeable.

Reply:
I do think "zephyr tree" is an upstream term that we should perhaps avoid. Instead we could potentially refer to it as the "main Zephyr repository". I will make that change.